app/components/createContainer/createContainerController.js

```diff
@@ -238,8 +236,8 @@ function ($scope, $state, $stateParams, $filter, Config, Info, Container, Contai
     var containerName = container;
     if (container && typeof container === 'object') {
       containerName = $filter('trimcontainername')(container.Names[0]);
-      if ($scope.swarm_mode) {
-        containerName = $filter('swarmcontainername')(containerName);
+      if ($scope.swarm_mode && containerName.indexOf('/') != -1) {
```

Comment:
The test here should be `if ($scope.swarm && !$scope.swarm_mode)` which would be triggered only with Docker Swarm (Swarm detection algorithm should be reviewed and will be updated later, it's ok for now).